tests/unit/contract_info_test.py

```
@@ -307,3 +326,119 @@ def test_proto_conversion_minimal_fields():
+            decline_reward=True,
+        ),
+    )
+    
+    contract_info = ContractInfo._from_proto(proto)
+    
+    assert contract_info.staked_account_id is None
+    assert contract_info.staked_node_id == 3
+    assert contract_info.decline_staking_reward is True
+
+
+def test_to_proto_with_staked_account_id(token_relationship):
```

Ruff (0.14.14)

[warning] Unused function argument: `token_relationship`

(ARG001)